tests/parser_tests/block_format_test.py

```diff
@@ -447,6 +450,84 @@ async def test_multiple_blocks(
     assert content == expected_content
 
 
+@pytest.mark.asyncio
+async def test_inverse(
+    mock_stream,
+    mock_call_llm_api,
+    mock_collect_user_input,
+    mock_setup_api_key,
+    mock_code_file_manager,
+    mock_git_root,
+):
+    # This test verifies that the inverse is a left inverse to stream_and_parse_llm_response. That is we can go:
+    #                               llm_message -> file_edits -> llm_message
+    # and get back where we started. Actually what we care about is that it's a right inverse. That if we go:
+    #                               file_edits -> llm_message -> file_edits
+    # we get back where we started. So this test verifies things we don't necessarily care about like the order of the
+    # edits and white space.
+    cwd = Path(os.getcwd())
```

Comment:
This will get an absolute path; normally the model outputs relative to the git_root (set to temp_testbed by default); not a big deal but worth pointing out